cmd/configschema/go.mod

```diff
@@ -279,6 +279,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dockerstatsreceiver v0.41.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dotnetdiagnosticsreceiver v0.41.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.41.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filereceiver v0.41.0 // indirect
```

Comment:
This shouldn't have the same version as I understand since it should be part of the next release.

Reply:
This version will change to match the current version when it's time to merge, no worries there. It's not merging for some time.